native/c/commands/ds.cpp

```diff
@@ -380,8 +380,9 @@ int handle_data_set_list(InvocationContext &context)
         {
           fields.push_back(it->dsorg);
           fields.push_back(it->volser);
-          fields.push_back(it->migr ? "true" : "false");
+          fields.push_back(it->migrated ? "true" : "false");
           fields.push_back(it->recfm);
+          // TODO Add more attributes here
```

Comment:
do we plan to add the new attributes as part of the `--rfc` fields here?  Or in a new PR? 😅